src/data/types/fragments.generated.tsx

```diff
@@ -0,0 +1,166 @@
+import { TypedDocumentNode as DocumentNode } from "@graphql-typed-document-node/core";
```

Comment:
Not a problem for today, but I am fairly strongly against having any generated sources committed to source control. Doing so suggests a defect in the build process.